es-client.yaml

```diff
@@ -39,32 +35,30 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
-        - name: "CLUSTER_NAME"
-          value: "myesdb"
+        - name: CLUSTER_NAME
+          value: myesdb
         - name: NODE_MASTER
           value: "false"
         - name: NODE_DATA
           value: "false"
         - name: HTTP_ENABLE
           value: "true"
```

Comment:
Can't we remove the double quotes from boolean values?

Reply:
alas no.
the value muse be a string - true, false and numbers are not strings.